doc/manual/src/plugins.md

```diff
@@ -0,0 +1,277 @@
+# Plugins
+
+This chapter describes all plugins present in hydra
+
+### Inputs
+
+Hydra supports the following inputs
+
+- Bazaar input
+- Darcs input
+- Git input
+- Mercurial input
+- Path input
+
+## Bitbucket pull requests
+
+Create jobs based on open bitbucket pull requests
+
+configuration options:
+- `bitbucket_authorization.<owner>`
+
+## Bitbucket status
+
+Sets Bitbucket CI status
+
+configuration options:
+- `enable_bitbucket_status`
+- `bitbucket.username`
+- `bitbucket.password`
+
+## Circle CI Notifcation
+
+Sets Circle CI status
+
+configuration options:
+- `circleci.[].jobs`
+- `circleci.[].vcstype`
+- `circleci.[].token`
+
+## Compress build logs
+
+Compresses build logs after a build with bzip2
+
+### Configuration options
+
+- `compress_build_logs`
+
+Enable log compression
+
+### Example
+
+```xml
+compress_build_logs = 1
+```
+
+## Coverity Scan
+
+Uploads source code to [coverity scan](https://scan.coverity.com)
+
+configuration options:
+
+- `coverityscan.[].jobs`
+- `coverityscan.[].project`
+- `coverityscan.[].email`
+- `coverityscan.[].token`
+- `coverityscan.[].scanurl`
+
+## Declarative jobsets
+
+TODO
+
+## Email notification
+
+Sends email notification if build status changes
+
+configuration options:
+
+- `email_notification`
+
+## Gitea status
+
+Sets Gitea CI status
+
+configuration options:
+
+- `gitea_authorization.<repo-owner>`
+
+## Github pulls
+
+Create jobs based on open github pull requests
```

Comment:
(product name)

```suggestion
## GitHub pulls

Create jobs based on open GitHub pull requests
```

There are several other capitalizations of GitHub below (GitHub, Github, github) and they should probably all be GitHub, except for `code blocks` and URLs.